spec/support/metadata_api_helpers.rb

```diff
@@ -97,6 +110,39 @@ def metadata_api_response_for_multipart_artefact
         { "web_url" => response["artefact"]["web_url"] + "/part-2" },
         { "web_url" => response["artefact"]["web_url"] + "/part-3" },
       ]
+      response["performance"]["page_views"] += [
+        {"value"=>25000,
+         "path"=>"/apply-uk-visa/part-1",
```

Comment:
to be closer to a real example, it's worth adding some example data for `"path"=>"/apply-uk-visa"` here, and the same applies further down for `response["performance"]["searches"]` and `response["performance"]["problem_reports"]`